hdf5-sys/src/h5i.rs

```diff
@@ -8,20 +8,27 @@ pub enum H5I_type_t {
     H5I_UNINIT = -2,
     H5I_BADID = -1,
     H5I_FILE = 1,
-    H5I_GROUP = 2,
-    H5I_DATATYPE = 3,
-    H5I_DATASPACE = 4,
-    H5I_DATASET = 5,
-    H5I_ATTR = 6,
+    H5I_GROUP,
+    H5I_DATATYPE,
+    H5I_DATASPACE,
+    H5I_DATASET,
+    #[cfg(hdf5_1_12_0)]
```

Comment:
This all starts to look fairly fragile, did they renumber existing enums? 🤦

Reply:
They added and removed members here. This caused some very odd errors that was hard to find...